contrib/wmenu/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "wmenu"
+pkgver = "0.1.5"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+    "scdoc",
+]
+makedepends = [
+    "cairo",
```

Comment:
you mean cairo-devel?

it ends up working because pango-devel pulls cairo-devel, but this needs a -devel on the end